src/js/views/wrappers/clickable.js

```diff
@@ -9,14 +9,19 @@ module.exports = Component => {
             };
         }
 
-        toggleClicked() {
+        toggleClicked(clickedNode) {
+            // Don't toggle the row if a link was clicked.
+            if (clickedNode.target.nodeName === 'A') {
+                return true;
+            }
+
             this.setState({
                 clicked: !this.state.clicked,
             });
         }
 
         render() {
-            return <Component {...this.props} {...this.state} toggleClicked={() => this.toggleClicked()} />;
+            return <Component {...this.props} {...this.state} toggleClicked={clickedNode => this.toggleClicked(clickedNode)} />;
```

Comment:
Change the `clickedNode` variable to `event`, because it's not a node it's an event. (Correct me if I'm wrong)